be/src/cloud/cloud_cumulative_compaction.cpp

```diff
@@ -405,6 +411,20 @@ void CloudCumulativeCompaction::process_old_version_delete_bitmap() {
                                         cloud_tablet()->tablet_id());
                             });
             if (update_st.ok()) {
+                std::vector<std::tuple<std::string, uint64_t, uint64_t>> to_delete;
+                for (auto it = new_delete_bitmap->delete_bitmap.begin();
+                     it != new_delete_bitmap->delete_bitmap.end(); it++) {
+                    to_delete.emplace_back(std::make_tuple(std::get<0>(it->first).to_string(),
+                                                           std::get<2>(it->first),
+                                                           std::get<2>(it->first)));
+                }
+                auto st = _engine.meta_mgr().remove_old_version_delete_bitmap(_tablet->tablet_id(),
```

Comment:
You shoud do the remove operation in the same FDB transaction with the update operation.